libretroshare/src/rsitems/rsdiscovery2items.h

```diff
@@ -156,38 +160,38 @@ class RsDiscIdentityListItem: public RsDiscItem
 
     std::list<RsGxsId> ownIdentityList ;
 };
-#if 0
-class RsDiscServicesItem: public RsDiscItem
+
+struct RsGossipDiscoveryInviteItem : RsDiscItem
```

Comment:
I dont really understand that name. Why the "Gossip" thing? YOure just sending a certificate here, right?

Reply:
Gossip discovery is the name that the whole service should take at some point, just `Disc` is too generic and create confusion with other possible method of peer discover, such as DHT, broadcast etc... Gossip apply well here because RsDisc works just like gossip (=people that say things about common friend each other) https://en.wikipedia.org/wiki/Gossip_protocol

Reply:
But yes that item just pass an invite on request

Reply:
RsDiscoveryCertificateItem and RsDiscoveryCertificateRequestItem would make more sense I would say.